src/macaron/dependency_analyzer/dependency_resolver.py

```diff
@@ -46,20 +52,19 @@ def __init__(self, resources_path: str, file_name: str, debug_path: str, tool_ve
             The path to the resources directory.
         file_name : str
             The name of dependency output file.
-        debug_path : str
-            The file path where all the dependencies will be stored for debugging.
+        tool_name: str
+            The name of the dependency analyzer.
         tool_version : str
             The version of the dependency analyzer.
+        repo_path: str
+            The path to the target repo.
```

Comment:
The documentation for `visited_deps` is missing.

Reply:
I didn't add `visited_deps` because it's not passed to the constructor as a parameter.

Reply:
Oops, I didn't look properly at the constructor's signature. All good.